lib/routes/content.js

```diff
@@ -0,0 +1,237 @@
+'use strict';
+
+/*
+ * @fileoverview Content system routing
+ *
+ */
+
+const config = require('config');
+const content = require('punchcard-content-types');
+const multipart = require('connect-multiparty');
+const uuid = require('uuid');
+
+const utils = require('../utils');
+const db = require('../database');
+
+const multipartMiddleware = multipart();
+
+/*
+ * Content Route Resolution
+ *
+ * @param {object} application - Express Application
+ * @returns {object} - Configured Express Application
+ */
+const contentRoutes = application => {
+  return new Promise(resolve => {
+    const app = application;
+    const types = app.get('content-types');
+
+    /*
+     * Content Home Page
+     */
+    app.get(`/${config.content.base}`, (req, res) => {
+      res.render('content-types/home', {
+        content: {
+          home: config.content.home,
+          base: config.content.base,
+          types,
+        },
+      });
+    });
+
+    /*
+     * Individual Content Type Landing Page
+     */
+    app.get(`/${config.content.base}/:type`, (req, res, next) => {
+      const type = utils.singleItem('id', req.params.type.toLowerCase(), types);
+
+      if (type === false) {
+        return next(new Error(`Content Type ${req.params.type} not found`));
```

Comment:
This error message we're going to want to be i18n friendly, so we don't want to hard-code it.